ERROR - 2025-06-18 13:43:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'landing'@'localhost' (using password: YES) in C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'landing', Object(SensitiveParameterValue), 'landing', 3306, '', 0)
#1 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rewardSystem\app\Filters\LanguageFilter.php(40): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Filters\Filters.php(241): App\Filters\LanguageFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#6 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore(Array)
#7 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('', 'before')
#8 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rewardSystem\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\rewardSystem\app\Filters\LanguageFilter.php(40): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Filters\Filters.php(241): App\Filters\LanguageFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#5 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore(Array)
#6 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('', 'before')
#7 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rewardSystem\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rewardSystem\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-06-18 13:43:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `admin_settings`
WHERE `setting_key` = :setting_key:', [...], false)
 3 APPPATH\Filters\LanguageFilter.php(40): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Filters\Filters.php(241): App\Filters\LanguageFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 5 SYSTEMPATH\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore([...])
 6 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('', 'before')
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 14:15:42 --> ErrorException: Undefined variable $music_settings
[Method: GET, Route: /]
in APPPATH\Views\public\landing_page.php on line 693.
 1 APPPATH\Views\public\landing_page.php(693): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $music_settings', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\public\\landing_page.php', 693)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\public\\landing_page.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('public/landing_page', [], true)
 5 APPPATH\Controllers\LandingController.php(70): view('public/landing_page', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LandingController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LandingController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))CRITICAL - 2025-06-18 14:27:41 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: reward]
in APPPATH\Views\public\reward_system.php on line 256.
 1 APPPATH\Views\public\reward_system.php(256): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\public\\reward_system.php', 256)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\public\\reward_system.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('public/reward_system', [], true)
 5 APPPATH\Controllers\RewardController.php(43): view('public/reward_system', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RewardController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RewardController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-18 15:05:10 --> processAutoClaim: Failed to insert claim into database
ERROR - 2025-06-18 15:05:10 --> processAutoClaim: Model errors: {"phone_number":"Phone number must be at least 10 digits"}
ERROR - 2025-06-18 15:05:13 --> processAutoClaim: Failed to insert claim into database
ERROR - 2025-06-18 15:05:13 --> processAutoClaim: Model errors: {"phone_number":"Phone number must be at least 10 digits"}
ERROR - 2025-06-18 15:06:46 --> Claim failed: No winner data
CRITICAL - 2025-06-18 15:49:33 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:49:35 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:49:36 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:53:26 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:56:35 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(157): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:58:25 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:02 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:02 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:02 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:02 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:03 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:34 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:35 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(146): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 15:59:51 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 16:00:11 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 16:00:12 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: user/update-background]
in SYSTEMPATH\BaseModel.php on line 1014.
 1 SYSTEMPATH\BaseModel.php(1014): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\UserController.php(149): CodeIgniter\Model->update('4', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->updateBackground()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:39:14 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\BonusClaimModel::getClaimsWithPagination
[Method: GET, Route: admin/bonus]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Admin\BonusController.php(40): CodeIgniter\Model->__call('getClaimsWithPagination', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:41:35 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\BonusClaimModel::getClaimsWithPagination
[Method: GET, Route: admin/bonus]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Admin\BonusController.php(40): CodeIgniter\Model->__call('getClaimsWithPagination', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:42:46 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\BonusClaimModel::getClaimsWithPagination
[Method: GET, Route: admin/bonus]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Admin\BonusController.php(40): CodeIgniter\Model->__call('getClaimsWithPagination', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:45:13 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\BonusClaimModel::getClaimsStats
[Method: GET, Route: admin/bonus]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Admin\BonusController.php(45): CodeIgniter\Model->__call('getClaimsStats', [])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:46:46 --> ErrorException: Undefined array key "total_claims"
[Method: GET, Route: admin/bonus]
in APPPATH\Views\admin\bonus\index.php on line 24.
 1 APPPATH\Views\admin\bonus\index.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "total_claims"', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php', 24)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('admin/bonus/index', [], true)
 5 APPPATH\Controllers\Admin\BonusController.php(62): view('admin/bonus/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:46:47 --> ErrorException: Undefined array key "total_claims"
[Method: GET, Route: admin/bonus]
in APPPATH\Views\admin\bonus\index.php on line 24.
 1 APPPATH\Views\admin\bonus\index.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "total_claims"', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php', 24)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('admin/bonus/index', [], true)
 5 APPPATH\Controllers\Admin\BonusController.php(62): view('admin/bonus/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:48:34 --> ErrorException: Undefined array key "today_claims"
[Method: GET, Route: admin/bonus]
in APPPATH\Views\admin\bonus\index.php on line 32.
 1 APPPATH\Views\admin\bonus\index.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "today_claims"', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php', 32)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('admin/bonus/index', [], true)
 5 APPPATH\Controllers\Admin\BonusController.php(62): view('admin/bonus/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:49:16 --> ErrorException: Undefined variable $today
[Method: GET, Route: admin/bonus]
in APPPATH\Models\BonusClaimModel.php on line 244.
 1 APPPATH\Models\BonusClaimModel.php(244): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $today', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Models\\BonusClaimModel.php', 244)
 2 APPPATH\Controllers\Admin\BonusController.php(45): App\Models\BonusClaimModel->getClaimsStats()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:50:07 --> ErrorException: Undefined array key "unique_ips"
[Method: GET, Route: admin/bonus]
in APPPATH\Views\admin\bonus\index.php on line 40.
 1 APPPATH\Views\admin\bonus\index.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "unique_ips"', 'C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php', 40)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\rewardSystem\\app\\Views\\admin\\bonus\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('admin/bonus/index', [], true)
 5 APPPATH\Controllers\Admin\BonusController.php(62): view('admin/bonus/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-18 18:51:06 --> CodeIgniter\Exceptions\BadMethodCallException: Call to undefined method App\Models\BonusClaimModel::getClaimsWithPagination
[Method: GET, Route: admin/bonus]
in SYSTEMPATH\Model.php on line 930.
 1 APPPATH\Controllers\Admin\BonusController.php(40): CodeIgniter\Model->__call('getClaimsWithPagination', [...])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\BonusController->index()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\BonusController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
